src/backend/controllers/AuthController.js

```diff
@@ -78,12 +78,12 @@ export const loginHandler = function (schema, request) {
       foundUser.password = undefined;
       return new Response(200, {}, { foundUser, encodedToken });
     }
-    new Response(
+    return new Response(
       401,
       {},
       {
         errors: [
-          "The credentials you entered are invalid. Unauthorized access error.",
+          "Incorrect Password!",
```

Comment:
We should not explicitly state which credentials are invalid

Reply:
Sure, I will correct those...